sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/tests/Helpers/ClientFactory.cs

```diff
@@ -0,0 +1,73 @@
+// ------------------------------------------------------------------------------------------------
```

Comment:
Copyright format are not consistent with our repo.

Reply:
Please use the format that is more prevalent in azure-sdk-for-net repo.

Reply:
Update files to match the format.